utils/cliutils/commandsflags.go

```diff
@@ -804,6 +808,14 @@ var flagsMap = map[string]cli.Flag{
 		Name:  archive,
 		Usage: "[Optional] Set to \"zip\" to deploy the files to Artifactory in a ZIP archive.` `",
 	},
+	uploadMinSplit: cli.StringFlag{
+		Name:  MinSplit,
+		Usage: "[Default: " + strconv.Itoa(UploadMinSplitMb) + "] Minimum file size in MB to split into ranges when uploading. Require Artifactory with S3 storage.` `",
+	},
+	uploadSplitCount: cli.StringFlag{
+		Name:  SplitCount,
+		Usage: "[Default: " + strconv.Itoa(UploadSplitCount) + "] Number of parts to split a file when uploading. Set to 0 for no splits.` `",
```

Comment:
Let's add the following to the end of the option description - 

This option, as well as the functionality of multi-part upload, requires Artifactory with S3 storage.